suite/tests/api/ir_aarch64.expect

```diff
@@ -839,23 +839,23 @@ sys    $0x1bd9 (%x29)[1byte]
 sys    $0x1bf1 (%x30)[1byte]
 sys    $0x1ba9 (%x1)[1byte]
 test_sys complete
-ldxr   (%x1)[8byte] $0x000000000000001f $0x000000000000001f -> %x0
-ldxrb  (%x1)[1byte] $0x000000000000001f $0x000000000000001f -> %w0
-ldxrh  (%x1)[2byte] $0x000000000000001f $0x000000000000001f -> %w0
-ldxp   (%x2)[8byte] $0x000000000000001f -> %w0 %w1
-ldxp   (%x2)[16byte] $0x000000000000001f -> %x0 %x1
-ldaxr  (%x1)[8byte] $0x000000000000001f $0x000000000000001f -> %x0
-ldaxrb (%x1)[1byte] $0x000000000000001f $0x000000000000001f -> %w0
-ldaxrh (%x1)[2byte] $0x000000000000001f $0x000000000000001f -> %w0
-ldaxp  (%x2)[8byte] $0x000000000000001f -> %w0 %w1
-stxr   %x0 $0x000000000000001f -> (%x1)[8byte] %w2
-stxrb  %w0 $0x000000000000001f -> (%x1)[1byte] %w2
-stxrh  %w0 $0x000000000000001f -> (%x1)[2byte] %w2
+ldxr   (%x1)[8byte] -> %x0
+ldxrb  (%x1)[1byte] -> %w0
+ldxrh  (%x1)[2byte] -> %w0
+ldxp   (%x2)[8byte] -> %w0 %w1
+ldxp   (%x2)[16byte] -> %x0 %x1
+ldaxr  (%x1)[8byte] -> %x0
+ldaxrb (%x1)[1byte] -> %w0
+ldaxrh (%x1)[2byte] -> %w0
+ldaxp  (%x2)[8byte] -> %w0 %w1
+stxr   %x0 -> (%x1)[8byte] %w2
+stxrb  %w0 -> (%x1)[1byte] %w2
+stxrh  %w0 -> (%x1)[2byte] %w2
 stxp   %w0 %w1 -> (%x2)[8byte] %w3
 stxp   %x0 %x1 -> (%x2)[16byte] %w3
-stlxr  %x0 $0x000000000000001f -> (%x1)[8byte] %w2
-stlxrb %w0 $0x000000000000001f -> (%x1)[1byte] %w2
-stlxrh %w0 $0x000000000000001f -> (%x1)[2byte] %w2
+stlxr  %x0 -> (%x1)[8byte] %w2
+stlxrb %w0 -> (%x1)[1byte] %w2
+stlxrh %w0 -> (%x1)[2byte] %w2
```

Comment:
There's no `smulh` or `umulh` here?  Do we expect that?  Maybe not everything is printed so those opcode changes in  `dis-a64.txt` are tested without any output here.

Reply:
I only updated the existing encoding tests, and I don't think smulh or umulh had any so they won't appear in this file. Dis-a64.txt doesn't use an expect to check its output